charm/sisyphus/actions/stop

```diff
@@ -0,0 +1,19 @@
+#!/usr/local/sbin/charm-env python3
+
+import sys
+import os
+import signal
+import sisyphus_utils
+
+from charms.layer.sisyphus import Sisyphus
+
+
+sisyphus = Sisyphus()
+if not sisyphus.is_running():
+    sisyphus_utils.fail('sisyphus is not running on this unit')
+
+os.kill(sisyphus.get_pid(), signal.SIGTERM)
```

Comment:
Needs to deal with the case where the pid in the lock file is no longer running:
```
>>> os.kill(99999999, 15)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ProcessLookupError: [Errno 3] No such process
```

Reply:
@cmars It does a check on line 12 to check whether it is running